app/src/main/java/com/example/riden/models/Ride.java

```diff
@@ -27,14 +101,70 @@ public int getSeats() {
         return getInt(SEATS_KEY);
     }
 
+    public void setSeats(int seats) {
+        put(SEATS_KEY, seats);
+    }
+
     public ParseFile getCarImage() {
+        try {
+            fetchIfNeeded().getString(CAR_IMAGE_KEY);
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
         return getParseFile(CAR_IMAGE_KEY);
     }
 
+    public void setCarImage(ParseFile image) {
+        put(CAR_IMAGE_KEY, image);
+    }
+
     public Boolean isReserved() {
         return getBoolean(IS_RESERVED_KEY);
```

Comment:
What does a ride being reserved mean? Should a seat have reservation instead? Can you calrify what happens if one rider reserves a seat do you decrement the number of seats. If so could we update the Ride object to have seats and available seats as 2 different parameters ?

Reply:
If a ride is reserved, then this means a rider has asked to request the ride. If a rider reserves a seat, I do decrement the number of seats. I will work on updating this with the ride object.